src/plugins/data/public/index_patterns/index_patterns/index_pattern.tsx

```diff
@@ -226,7 +221,7 @@ export class IndexPattern implements IIndexPattern {
             <p>{warningText}</p>
             <EuiFlexGroup justifyContent="flexEnd" gutterSize="s">
               <EuiFlexItem grow={false}>
-                <EuiButton size="s" href={kbnUrl.getRouteHref(this, 'edit')}>
+                <EuiButton size="s" href={editUrl}>
```

Comment:
![Screen Shot 2019-12-02 at 1 42 24 PM](https://user-images.githubusercontent.com/16738889/69953398-361dcb80-150a-11ea-910d-6bf777a48b59.png)
 Unfortunately, a base path is missed here and link follows to not exist url: `/management/kibana/index_patterns/d3d7af60-4c81-11e8-b3d7-01146121b73d`

Reply:
@Avinar-24 , can you clone this PR and make the required change?

Reply:
Fixed